disnake/emoji.py

```diff
@@ -184,27 +187,30 @@ def guild(self) -> Optional[Guild]:
 
     @property
     def application_id(self) -> Optional[int]:
-        """Optional[:class:`int`]: The ID of the application which owns this emoji.
+        """Optional[:class:`int`]: The ID of the application which owns this emoji,
```

Comment:
This should always be ourselves.

Reply:
Hmm, thoughts on removing this property entirely?

Reply:
Since this property only pulls from `ConnectionState.application_id` and doesn't actually depend on the emoji's data, I don't think it's super useful in its current form